mirny.py

```diff
@@ -153,6 +153,40 @@ def connect_ext(self, ext, adr, mask):
             ),
         ]
 
+    def connect_almazny_passthrough(self, pt, adr, mask):
+        # for use w/ almazny
+        self._check_intersection(adr, mask)
+        self._slaves.append((pt, adr, mask))
+        stb = AsyncRst()
+        self.submodules += stb
+
+        saved_adr = Signal(2)
+        latch_signal = Signal()
+
+        self.comb += [
+            stb.i.eq(((self.bus.adr & adr) == adr) & (self.ext.cs)),
+            stb.ce.eq(self.bus.re),
+            pt.mosi.eq(self.ext.sdi & stb.o),
+            pt.clk.eq(self.ext.sck & stb.o),
```

Comment:
Same as L148. Probably no point in blanking this with `stb`.

Reply:
To be honest, even the clock could be unblanked - as for these shift registers, only last 8 data cycles ended with latch rising up matter. But yes, you're right - there's not much reason except maybe cleaner waveform signal (probably why I did it when I was debugging this code with logic analyzer).

Reply:
No. **Especially the clock** should be unblanked. That's where my comment is. Just follow `connect_ext`.